ignite/src/main/java/org/hibernate/ogm/datastore/ignite/PortableBuilderDelegate.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.ignite;
+
+import org.apache.ignite.binary.BinaryObjectException;
+import org.jetbrains.annotations.Nullable;
+
+public interface PortableBuilderDelegate {
```

Comment:
Hello!
I think I've done all you asked.
I removed criteria implementation, unused classes and GridGain support.

Victor


2016-03-17 12:43 GMT+03:00 Victor Z <vikad111@gmail.com>:

> We don't need it now. I'll remove it.
>
> 2016-03-17 3:01 GMT+03:00 DavideD <notifications@github.com>:
>
>> In
>> ignite/src/main/java/org/hibernate/ogm/datastore/ignite/PortableBuilderDelegate.java
>> <https://github.com/hibernate/hibernate-ogm/pull/628#discussion_r56436981>
>> :
>>
>> > @@ -0,0 +1,81 @@
>> > +/*
>> > + * Hibernate OGM, Domain model persistence for NoSQL datastores
>> > + *
>> > + * License: GNU Lesser General Public License (LGPL), version 2.1 or later
>> > + * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
>> > + */
>> > +package org.hibernate.ogm.datastore.ignite;
>> > +
>> > +import org.apache.ignite.binary.BinaryObjectException;
>> > +import org.jetbrains.annotations.Nullable;
>> > +
>> > +public interface PortableBuilderDelegate {
>>
>> THis is not use now, Is it a replacement for the BinaryObjectBuilder?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/hibernate/hibernate-ogm/pull/628/files/725f0a5f26036d1ab91f36cd4cb901d0675e0c12#r56436981>
>>
>
>